tests/unittests/test_dump.py

```diff
@@ -232,3 +233,9 @@ def test_dumping_indexset():
 
 def test_dumping_selfassign():
     assert SelfAssign(1)._pd_dumps() == read_dump('selfassign')
+
+
+def test_othermodule_function():
+    x = tim.function._pd_dumps()
+    # TODO: wip
+    #  assert 'todo' == 'done'
```

Comment:
Should this assert be uncommented? Or left as is for moment?

Reply:
No it should be left as it is.